DeAceroDLabs/marketplace-frontend
# Feature/dmp 550  Carousel component

## What is done

[[DMP-550] Carousel component](https://deacero.atlassian.net/browse/DMP-550)

## How is done

- Created a `Carousel` component that will be used  to show all categories on home page and to show the product images on ProductDetail view.
- This Carousel now receives two types: "images" and "tabs".
- Added `@react-hook/media-query` module to change number of visible elements inside carousel depending on window size.

## How to test

1. Run the project via `npm start`
2. npm i
3. You should see the Home Screen with categories carrousel. Play with it by clicking on categories and arrow buttons.
4. To view the `@react-hook/media-query` usage, please change the window size with development tools.
5. /producto/1
6. You should see the images carrousel.
